src/main/java/uk/gov/cshr/civilservant/service/OrganisationalUnitService.java

```diff
@@ -164,11 +166,23 @@ public AgencyTokenResponseDto getAgencyToken(Long organisationalUnitId) throws C
 
     @Cacheable("organisationalUnitsTree")
     public List<OrganisationalUnit> getOrgTree() {
-        return this.getParents();
+        List<OrganisationalUnit> listOrg = this.getParents();
+        sortOrganisationList(listOrg);
+        return listOrg;
     }
 
     @Cacheable("organisationalUnitsFlat")
     public List<OrganisationalUnitDto> getFlatOrg() {
         return this.getListSortedByValue();
     }
+
+    private void sortOrganisationList(List<OrganisationalUnit> list) {
+        list.forEach(org ->
+            {
+                if(org.hasChildren())
+                    org.getChildren().sort(Comparator.comparing(OrganisationalUnit :: getName, String.CASE_INSENSITIVE_ORDER)) ;
+                    sortOrganisationList(org.getChildren());
```

Comment:
This is the recursive call so could you please add the comment around this code to explain how the execution of this code will come out this recursive call.